types/api/api.d.ts

```diff
@@ -0,0 +1,9 @@
+export declare class API {
+    constructor(kwargs?: Record<string, unknown>)
+    get(url: string, options?: Record<string, unknown>): any
```

Comment:
I'll accept this one because it's an improvement over what we had but `option` should be typed much more strongly based on its usages. Using an arbitrary object doesn't help at all, but knowing that I can pass `params`/`handle`/etc serves as powerful documentation.

Please revisit this as soon as you can.

Reply:
On it